static/docs/commands-reference/repro.md

```diff
@@ -98,6 +98,13 @@ local cache and updates stage files with the new checksum information.
 
 - `--downstream` - rerun the commands present in the downstream of the pipeline.
 
+- `R, --recursive` - the `targets` value is expected to be a directory path.
+  With this option, you can reproduce all stages in a specified directory. 
+  Determines the files to download by searching the named directory and its
+  subdirectories for DVC files to download data for. Along with providing a 
```

Comment:
DVC-files, as we discussed with @jorgeorpinel we will be using this format

Reply:
Ok

Reply:
I'll double check when fixing #321 anyway but yes thanks!